src/service/action-impl.js

```diff
@@ -646,6 +646,16 @@ export function parseActionMap(s, context) {
 
   let actionMap = null;
 
+  const isStandaloneStory = !!closest(context, el => {
+    return el.classList.contains('i-amphtml-story-standalone');
```

Comment:
Possible race condition since this requires `amp-story` extension to be downloaded and built.